TicketSwap/solar
# Introduce `StylisedIcon` & `Prompt` components

# Description

In the new cover, but also in this new Prompt component we use an icon on a circle background. Because of this and after aligning with Mark I created a new component called `StylisedIcon` which can be used for the Cover component but also for the early mentioned Prompt component that is being introduced in this PR as well. The Prompt component can serve as empty state, but later on perhaps also as error state. Because both only use a default theming I didn't introduce variants yet, but that shouldn't be hard to setup as soon as we need them.

## How to test

- Checkout this branch
- `yarn storybook`
- Check the stories of `Prompt` and `StylisedIcon`
- Check the stories of `CoverV2` and verify it looks like expected

## To be notified

@Marruk
